projects/canopy/src/lib/breadcrumb/breadcrumb.stories.ts

```diff
@@ -1,127 +1,153 @@
-import { Component, OnInit, Input } from '@angular/core';
-
-import { select, withKnobs } from '@storybook/addon-knobs';
-import { moduleMetadata } from '@storybook/angular';
+import { Meta, moduleMetadata, Story } from '@storybook/angular';
 
+import { BreadcrumbVariant, LgBreadcrumbComponent } from '.';
 import { LgIconModule } from '../icon/icon.module';
-import { BreadcrumbVariant } from './breadcrumb-item/breadcrumb-item.interface';
+// import { BreadcrumbVariant } from './breadcrumb-item/breadcrumb-item.interface';
 import { LgBreadcrumbModule } from './breadcrumb.module';
 import { notes } from './breadcrumb.notes';
 
-interface Breadcrumb {
-  text: string;
-}
-
-@Component({
-  selector: 'async-breadcrumb-story',
-  template: `
-    <lg-breadcrumb [variant]="variant">
-      <lg-breadcrumb-item *ngFor="let crumb of crumbs; index as i">
-        <a href="#" [attr.aria-current]="isCurrentAttribute(i)">
-          <lg-icon *ngIf="!i" [name]="'home'"></lg-icon>
-          {{ crumb.text }}
-        </a>
-      </lg-breadcrumb-item>
-    </lg-breadcrumb>
-  `,
-})
-export class AsyncBreadcrumbStoryComponent implements OnInit {
-  @Input() variant: BreadcrumbVariant;
-
-  crumbs: Array<Breadcrumb> = [];
-
-  currentPageIndex = this.crumbs.length - 1;
-
-  isCurrentAttribute(index: number) {
-    return this.currentPageIndex === index ? 'page' : null;
-  }
-
-  ngOnInit() {
-    const mockAsyncDelay = 500;
-
-    setTimeout(() => {
-      this.crumbs = [
-        { text: 'Home' },
-        { text: 'Home Insurance' },
-        { text: 'Change direct debit' },
-      ];
-    }, mockAsyncDelay);
-  }
-}
+// interface Breadcrumb {
```

Comment:
Do you need this anymore?

Reply:
Ah, forgot about that, cheers Paul. Removed now ✅